scala3/scaladoc.md

```diff
@@ -47,7 +47,7 @@ The following features are currently (May 2021) not stable to be released with s
 
 ### Snippet compiling
 
-One of the experimental features of Scaladoc is a snippets compiler. This tool will allow you to compile snippets that you attach to your docstring
+One of the experimental features of Scaladoc is a snippets' compiler. This tool will allow you to compile snippets that you attach to your docstring
```

Comment:
this is incorrect. "compiler for snippets" would be an acceptable rewording, but I also think the original was fine